src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterEnterOnTokenTests.cs

```diff
@@ -978,6 +978,280 @@ class What
                 expectedIndentation: 8);
         }
 
+        [Fact]
+        [WorkItem(872)]
+        [Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        public void InsideInterpolationString_1()
+        {
+            var code = @"class Program
```

Comment:
All our test code uses this. Moreover this is not going to improve pref by any considerable amount. So I am going to leave it.